File: server/model/notification.go
package model

import "time"

type Notification struct {
	PageURL    string    `json:"page_url"`
	Selector   string    `json:"selector"`
	Text       string    `json:"text"`
	WebHookURL string    `json:"webhook_url"`
	Timestamp  time.Time `json:"time,omitempty"`
}

File: web/Models/Notification.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Kumo.Models {
  public class Notification {
    [Url]
    [Required]
    [JsonPropertyName ("webhook_url")]
    public string WebHookUrl { get; set; }

    public Notification () {
      WebHookUrl = "";
    }
  }
}
